spec/dor/services/client/workspace_spec.rb

```diff
@@ -38,16 +38,22 @@
   end
 
   describe '#cleanup' do
-    subject(:request) { client.cleanup }
+    subject(:request) { client.cleanup(workflow: workflow, lane_id: lane_id) }
+
+    let(:workflow) { nil }
+    let(:lane_id) { nil }
```

Comment:
Does it make sense for these two `let`s to be moved into the `when API request fails` context?

Reply:
That would be perfectly reasonable. However, I tend to leave default `let`s on the higher level because it makes it clear that those `let`s only need to be provided when it is relevant for the test.

If you look at our specs, I think you'll see both practices.